board/pinecube/os.conf

```diff
@@ -0,0 +1 @@
+OS_AP="wlan0"
```

Comment:
Are you sure the AP interface name here sould be `wlan0`? Generally, the kernel will use `wlan0` to identify the Wi-Fi network card and generate an `ap0` interface.

Reply:
I am not 100 % sure how AP should work. With default settings (`ap0`), it fails with following lines:
```
 * Starting hostapd on ap0: done
 * Starting dnsmasq: SIOCSIFADDR: No such device
ap0: ERROR while getting interface flags: No such device

```
When I change it to `wlan0`, it works. rtl8189es is out-of-tree driver, so who knows what behavior we can expect.
I also see driver logging following lines (among many others) - maybe it fails to generate `ap0` interface?
[   25.790226] RTW: cfg80211_rtw_add_virtual_intf(phy0) name:ap0, type:3
[   25.796709] RTW: WARN adapter pool empty!
[   25.800840] RTW: cfg80211_rtw_add_virtual_intf(phy0) wdev:00000000, ret:-19